src/stories/button/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { type ReactNode } from 'react';
+import './styles/index.css';
+import { type IButton } from './types';
+
+function Button({
+  children,
+  loading = false,
+  ariaLabel,
+  type,
+  startIcon,
+  endIcon,
+  variant,
+  className,
+  href,
+  target,
+  onClick,
+  style,
+  ...rest
+}: IButton): ReactNode {
```

Comment:
Add hover effect also for button and make button similar to the UI in nike website